docs/user-guide/nginx-configuration/annotations.md

```diff
@@ -191,10 +191,16 @@ nginx.ingress.kubernetes.io/auth-realm: "realm string"
 
 NGINX supports load balancing by client-server mapping based on [consistent hashing](http://nginx.org/en/docs/http/ngx_http_upstream_module.html#hash) for a given key. The key can contain text, variables or any combination thereof. This feature allows for request stickiness other than client IP or cookies. The [ketama](http://www.last.fm/user/RJ/journal/2007/04/10/392555/) consistent hashing method will be used which ensures only a few keys would be remapped to different servers on upstream group changes.
 
+There is a special mode of upstream hashing called subset. In this mode, upstream servers are grouped into subsets, and stickness works by mapping keys to a subset instead of individual upstream servers. It provides a balance between stickness and load distribution.
```

Comment:
s/stickness/stickiness

Reply:
It would also be nice to mention that a specific server is chosen uniformly at random from the selected sticky subset.

Reply:
Done.